pylint/utils/utils.py

```diff
@@ -398,3 +407,25 @@ def _ini_format(stream, options):
                 # remove trailing ',' from last element of the list
                 value = value[:-1]
             print("%s=%s" % (optname, value), file=stream)
+
+
+class IsortDriver:
+    """A wrapper around isort API that changed between versions 4 and 5."""
+
+    def __init__(self, known_third_party, known_standard_library):
+        if HAS_ISORT_5:
+            self.isort5_config = isort.api.Config(
+                known_third_party=known_third_party,
+                known_standard_library=known_standard_library,
```

Comment:
I think, you'll want to use `extra_standard_library` here if you the behaviour to stay the same as the isort4 version: https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/#known_standard_library